oaknational/oak-ai-lesson-assistant
# fix: update aila chat e2e test to handle 10 stages

## Description

- Fix playwright tests failing because we've reduced the stages from 12 to 10.

## How to test

1. Change the base branch to production, which should trigger all the playwright tests.
2. If it passes, then switch the base back to main before merging.
